MTM-Backend/src/types/donation.ts

```diff
@@ -19,3 +19,15 @@ export interface DonationDetailsType {
   quantityUsed: number;
   quantityNew: number;
 }
+
+export interface reqBodyType {
```

Comment:
Could you give a better name like DonationRequestBodyType

Reply:
Yep